### PR TITLE
Kkanas/documentation

### PR DESCRIPTION
**Pull request by kkanas**
## Small fixes discovered upon inspecting the documentation:

Add src/adapters/mc/OsConfigPolicy_DeployIfNotExists.json  that was deleted in ee235d2329eac5b123f732690ce119cccfb7a5fe as it is referenced by the documentation
Fix cmake version on ubuntu 14,
Fix small typos

## Checklist

- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [ ] I added unit-tests to validate my changes. All unit tests are passing.
- [ ] I have merged the latest `main` branch prior to this PR submission.
- [ ] I submitted this PR against the `main` branch.
## Checklist

- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [ ] I added unit-tests to validate my changes. All unit tests are passing.
- [ ] I have merged the latest `main` branch prior to this PR submission.
- [ ] I submitted this PR against the `main` branch.

### CODE REVIEW

File: src/adapters/mc/README.md [hunk 1]
```diff
@@ -15,16 +15,16 @@ To regenerate code, see [codegen.cmd](codegen.cmd).
...
 The OSConfig Universal NRP binary is built on Ubuntu 14 with gcc 4.8 in order for this same binary to run as-is on all newer Linux distros.
 
 Install Ubuntu 14.04 LTS from one of the archived locations such as at [Ubuntu](https://www.releases.ubuntu.com/14.04/), or [Ubuntu MATE](https://releases.ubuntu-mate.org/archived/14.04/amd64/)). 
 
-Install the miminal set of dependencies necessary:
+Install the minimal set of dependencies necessary:
 
 ```bash
-sudo apt-get install -y gcc git cmake build-essential
+sudo apt-get install -y gcc git cmake3 build-essential
```
Comment (MariusNi) [inline src/adapters/mc/README.md:27]: >cmake3 [](http://example.com/codeflow?start=32&length=6)

Actually, as it is documented, this is literally how we got it building on Ubuntu 14. Please do not do this one change.

File: src/adapters/mc/OsConfigPolicy_DeployIfNotExists.json [hunk 2]
```diff
@@ -0,0 +1,923 @@
+{
+    "properties": {
+        "displayName": "[Preview] SSH Posture Control policy",
+        "policyType": "Custom",
+        "mode": "Indexed",
+        "description": "This policy ensures that the SSH Server is securely configured on the Linux device",
+        "metadata": {
```
Comment (MariusNi) [inline src/adapters/mc/OsConfigPolicy_DeployIfNotExists.json:7]: Please do not re-add this back.

**Comment by MariusNi** [comment]
> An example of a generated policy definition JSON (with the SAS token and file hash removed) for the above example is available here: [OsConfigPolicy_DeployIfNotExists.json](OsConfigPolicy_DeployIfNotExists.json)

We need this changed to: ssh/LinuxSshServerSecurityBaseline_DeployIfNotExists.json

---
Refers to: src/adapters/mc/README.md:248 in aadf985. [](commit_id = aadf98503bdd8e47fe8786e4909e8cbf4b145cb5, deletion_comment = False)

File: src/adapters/mc/README.md (hunk 1 above)
Comment (kkanas) [reply src/adapters/mc/README.md:27]: When I tried to build it on ubuntu 14 it required cmake >= 3.2 
The default cmake ( 2.8.12.2) was upgraded if cmake3 package was installed 
You can check if you have cmake3 installed by running `dpkg -l|grep cmake3`

**Pull request closed without merging**